app/src/test/java/shop/woowasap/domain/product/DomainFixture.java

```diff
@@ -0,0 +1,19 @@
+package shop.woowasap.domain.product;
+
+import java.math.BigInteger;
+import java.time.Instant;
+import shop.woowasap.shop.domain.Product;
+
+public class DomainFixture {
+
```

Comment:
sonar rule : util성 클래스는 private 생성자를 정의하고 호출되었을때 메시지를 던져줘야 합니다 :)

```suggestion
private DomainFixture() {
	throw new UnsupportedOperationException("Cannot invoke constructor \"DomainFixture()\"");
}
```